next.config.js

```diff
@@ -1,22 +1,10 @@
 const withSvgr = require('next-svgr')
 const withBundleAnalyzer = require('@next/bundle-analyzer')
 const withPlugins = require('next-compose-plugins')
-const rehypeShiki = require('rehype-shiki')
 const checkEnv = require('@47ng/check-env').default
 const withImages = require('next-images')
 const withMDX = require('@next/mdx')({
   extension: /\.mdx?$/,
-  options: {
-    rehypePlugins: [
```

Comment:
This deletes `rehype-shiki`. Instead, we use [`prism-react-renderer`](https://github.com/FormidableLabs/prism-react-renderer), which provides the ability to manipulate what Prism renders on the UI (adding line numbering, line highlighting, and language syntaxes)

Reply:
I'm a fan of Shiki because it uses the tokenizer that VSCode uses, resulting in super fine grained highlighting. And you can still control the code block component, [Lorenzo](https://lorenzopepe.dev/blog/build-a-blog-2020) has been doing some great stuff with it: https://github.com/lorenzored98/lorenzopepe/blob/master/Components/Code/CodeBlock.tsx (line numbering and highlighting, etc) so perhaps we could borrow some of their ideas?

Reply:
The tradeoffs are valid. We can definitely borrow some of their ideas and not use prism. I'll close this PR and start from scratch. Thank you for taking the time to review this. 😄